src/code/fv-components/fv-chromosome-image.js

```diff
@@ -0,0 +1,33 @@
+import React, {PropTypes} from 'react';
+import classNames from 'classnames';
+
+const FVChromosomeImageView = ({small=false, bold=false, empty=false, yChromosome=false, animationStyling}) => {
+  let positionStyling = {};
```

Comment:
Copied and pasted from chromosome image, so some of this code is not used (eg animation styling)